lib/cf-app-utils/cf/app/credentials.rb

```diff
@@ -1,20 +1,26 @@
 module CF::App
   class Credentials
-    class << self
+
+      attr_reader :service_service
```

Comment:
I think the Service class is more like a ServiceLookup class that gives you back a Service.